ingest/core/query_test.py

```
@@ -0,0 +1,209 @@
+            results = vector_store.query_similar(query, top_k=3)
+            if results:
+                print("✅ Found {len(results)} results (top score: {results[0]['score']:.4f})")
+                # Print just the top result details
+                top_result = results[0]
+                top_result["metadata"]
+                print(
+                    "   Best match: {metadata.get('file_name', 'Unknown')} (chunk {metadata.get('chunk_index', 'Unknown')})"
+                )
+            else:
+                print("❌ No results found")
+        except Exception:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)